model_eval.py

```diff
@@ -170,7 +239,7 @@ def evaluate_model(self):
         # Placeholder for DEP code
         #elif self.task == 'dep':
 
-        return micro_f1, macro_f1
+        return micro_f1, macro_f1, uas, las

```

Comment:
This will throw an error for any task other than dep, so should be inside the if that has all the dep model training inside it.